lib/utils/key_value_repository.dart

```diff
@@ -17,11 +17,11 @@ class KeyValueRepository with LogMixin {
   }
 
   /// [value] must have a type that is supported by sembast
-  Future<dynamic> put(String key, dynamic value) async {
-    if (key == 'privateKeyKey') {
-      logger.info('Put setting with key "$key"');
-    } else {
+  Future<dynamic> put(String key, dynamic value, bool displayValue) async {
+    if (displayValue) {
       logger.info('Put setting with key "$key" and value "$value"');
+    } else {
+      logger.info('Put setting with key "$key"');
```

Comment:
I was thinking more like:
```dart
Future<dynamic> put(String key, dynamic value, {bool displayValue = true,}) async { ... }
```
This way we don't need to change the old code and only specify it `displayValue: false` where it is necessary.

Reply:
oh yes 😄